Tests failing in master

Two of the tests are failling in current master.
```
======================================================================
FAIL: pydy.codegen.tests.test_ode_function_generator.TestODEFunctionGeneratorSubclasses.test_rhs_args
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/sahil/anaconda/envs/pydy-dev/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/sahil/pydy/pydy/codegen/tests/test_ode_function_generator.py", line 362, in test_rhs_args
    np.testing.assert_allclose(xdot, last_xdot)
  File "/home/sahil/anaconda/envs/pydy-dev/lib/python2.7/site-packages/numpy/testing/utils.py", line 1297, in assert_allclose
    verbose=verbose, header=header)
  File "/home/sahil/anaconda/envs/pydy-dev/lib/python2.7/site-packages/numpy/testing/utils.py", line 665, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Not equal to tolerance rtol=1e-07, atol=0

(mismatch 100.0%)
 x: array([ 0.099968,  0.050154,  0.4285  ,  0.618715, -0.436786, -2.954496,
        0.790694,  0.003718])
 y: array([ 0.099968,  0.050154,  0.4285  ,  0.618715, -0.549453, -1.488752,
        0.501619,  0.026753])

======================================================================
FAIL: pydy.tests.test_utils.test_sympy_equal_to_or_newer_than
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/sahil/anaconda/envs/pydy-dev/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/sahil/pydy/pydy/tests/test_utils.py", line 9, in test_sympy_equal_to_or_newer_than
    assert sympy_equal_to_or_newer_than('0.7.6', '0.7.6-git')
AssertionError

----------------------------------------------------------------------

```

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,9 +2,11 @@ language: python
 matrix:
   include:
     - python: 2.7
-      env: SYMPY_MASTER="false"
+      env: SYMPY_VERSION="oldest"
     - python: 2.7
-      env: SYMPY_MASTER="true"
+      env: SYMPY_VERSION="latest"
+    - python: 2.7
+      env: SYMPY_VERSION="master"
 virtualenv:
   system_site_packages: true
 before_install:
@@ -21,9 +23,11 @@ install:
   # Theano may try to install SciPy from source if the --no-deps flags isn't
   # there.
   - pip install --no-deps Theano==0.6.0
-  - if [[ $SYMPY_MASTER == "false" ]]; then
+  - if [[ $SYMPY_VERSION == "oldest" ]]; then
       pip install sympy==0.7.4.1;
-    elif [[ $SYMPY_MASTER == "true" ]]; then
+    elif [[ $SYMPY_VERSION == "latest" ]]; then
+      pip install sympy;
+    elif [[ $SYMPY_VERSION == "master" ]]; then
       pip install https://github.com/sympy/sympy/archive/master.zip;
     fi
   - pip install -U nose

--- a/pydy/codegen/tests/test_ode_function_generator.py
+++ b/pydy/codegen/tests/test_ode_function_generator.py
@@ -282,8 +282,8 @@ class TestODEFunctionGeneratorSubclasses(object):
         # There are eight constants and four specified inputs available.
         sys = models.n_link_pendulum_on_cart(3, True, True)
         right_hand_side = sys.eom_method.rhs()
-        constants = sys.constants_symbols
-        specifieds = sys.specifieds_symbols
+        constants = list(sm.ordered(sys.constants_symbols))
+        specifieds = list(sm.ordered(sys.specifieds_symbols))
 
         constants_arg_types = [None, 'array', 'dictionary']
         specifieds_arg_types = [None, 'array', 'function', 'dictionary']
@@ -320,7 +320,7 @@ class TestODEFunctionGeneratorSubclasses(object):
                 g = LambdifyODEFunctionGenerator(right_hand_side,
                                                  sys.coordinates,
                                                  sys.speeds,
-                                                 sys.constants_symbols,
+                                                 constants,
                                                  specifieds=specifieds,
                                                  constants_arg_type=p_arg_type,
                                                  specifieds_arg_type=r_arg_type)
@@ -338,7 +338,7 @@ class TestODEFunctionGeneratorSubclasses(object):
         # Now make sure it all works with specifieds=None
         sys = models.n_link_pendulum_on_cart(3, False, False)
         right_hand_side = sys.eom_method.rhs()
-        constants = sys.constants_symbols
+        constants = list(sm.ordered(sys.constants_symbols))
 
         del last_xdot
 
@@ -348,7 +348,7 @@ class TestODEFunctionGeneratorSubclasses(object):
                 g = LambdifyODEFunctionGenerator(right_hand_side,
                                                  sys.coordinates,
                                                  sys.speeds,
-                                                 sys.constants_symbols,
+                                                 constants,
                                                  constants_arg_type=p_arg_type,
                                                  specifieds_arg_type=r_arg_type)
 

